language/vm/vm-runtime/src/libra_vm.rs

```diff
@@ -67,7 +48,7 @@ impl VMVerifier for LibraVM {
         // TODO: This should be implemented as an async function.
         record_stats! {time_hist | TXN_VALIDATION_TIME_TAKEN | {
             self.move_vm
-                .rent(move |runtime| verify_transaction(runtime, transaction, state_view))
+                .execute_runtime(move |runtime| verify_transaction(runtime, transaction, state_view))
```

Comment:
Can we add a TODO for renaming this in the future to avoid confusion with bytecode verification?